lib/index.js

```diff
@@ -16,30 +16,45 @@ var Yandex = module.exports = integration('Yandex Metrica')
   .assumesPageview()
   .global('yandex_metrika_callbacks')
   .global('Ya')
+  .option('type', 0)
   .option('counterId', null)
   .option('clickmap', false)
   .option('webvisor', false)
+  .option('trackHash', false)
+  .option('trackLinks', false)
+  .option('accurateTrackBounce', false)
+  .option('params', null)
```

Comment:
The best way would be to define a global object outside of segment that contains the parameters you want to set. Then when we initialize yandex, we can use that global variable.

https://github.com/segment-integrations/analytics.js-integration-sentry/blob/master/lib/index.js#L47

Can you explain what kind of things you'd want to set via this params object?